doc/source/getting_started/earthdata.rst

```diff
@@ -0,0 +1,40 @@
+=====
+NSIDC
+=====
+
+NASA has 12 `Distributed Active Archive Centers (DAACs) <https://earthdata.nasa.gov/about/daacs>`_
+throughout the United States.
+All ICESat-2 data is archived and stored at the NASA DAAC at the
+`National Snow and Ice Data Center (NSIDC) <https://nsidc.org/daac/>`_.
+``icepyx`` uses `NASA Common Metadata Repository (CMR) <https://cmr.earthdata.nasa.gov/search>`_
+queries to search for ICESat-2 granules that fit a given set of spatial, temporal or orbital parameters.
+These granules can then be requested and downloaded from the NSIDC archives.
+Granules returned by the CMR system may occasionally not spatially intersect the region of interest.
+This is due to a spatial margin (buffer) applied to the ICESat-2 CMR resources.
+
+Startup Steps for NSIDC
+#######################
+
+1. `Register with NASA Earthdata Login system <https://urs.earthdata.nasa.gov/users/new>`_
+2. Use your NASA Earthdata credentials either:
+
+   * Directly in an ``icepyx`` Notebook
+   * Permanently with a `.netrc file <https://nsidc.org/support/how/v0-programmatic-data-access-guide>`_
+
+    .. code-block:: bash
+
+        echo "machine urs.earthdata.nasa.gov login <uid> password <password>" >> ~/.netrc
+        chmod 0600 ~/.netrc
+
+   * Temporarily with an environmental variable before running ``icepyx``
+
+    .. code-block:: bash
+
+        export EARTHDATA_PASSWORD=<password>
```

Comment:
I think this is how people would set an environment variable on Windows, though haven't tested it properly.

```suggestion
        export EARTHDATA_PASSWORD=<password>  # on macOS/Linux
        set "EARTHDATA_PASSWORD=<password>"  # on Windows
```